test/irmin-bench/replay.ml

```diff
@@ -10,17 +10,16 @@ let check t = Alcotest.check (testable t)
 module Conf = Irmin_tezos.Conf
 
 module Store = struct
-  type store_config = string
+  type store_config = unit
 
   module Store = Irmin_tezos.Store
 
-  let create_repo store_dir =
-    let conf = Irmin_pack.config ~readonly:false ~fresh:true store_dir in
+  let create_repo ~root () =
```

Comment:
That `unit` comes from `repo_config` just above. It is `unit` in most cases, it could be something else if we wanted to control something specific from outside the replay. For example, it could carry this information: `let's launch a layered store freeze every 10k commits`.